lib/ckb/api.rb

```diff
@@ -51,6 +59,11 @@ def set_system_script_cell(out_point, code_hash)
       @system_script_code_hash = code_hash
     end
 
+    # @param out_point [CKB::Types::OutPoint]
+    def set_system_script_group_cell(out_point)
```

Comment:
The func name is no longer suitable. It's not setting a cell but an out point.

Reply:
this cell is not equal with the cell on CKB, but I will use another name. thanks.